src/main/java/uk/gov/laa/ccms/caab/bean/AddressSearchFormData.java

```diff
@@ -6,6 +6,6 @@
  * Represents the client address search form.
```

Comment:
Change javadoc now that class has been renamed?

Reply:
renamed